src/patternfly/components/Tile/tile.scss

```diff
@@ -0,0 +1,170 @@
+  --pf-c-tile--m-display-lg__header--m-stacked__icon--Height: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--Width);
+
+  // body
+  --pf-c-tile__body--Color: var(--pf-global--Color--100);
+  --pf-c-tile__body--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-tile--disabled__body--Color: var(--pf-global--disabled-color--100);
+
+  position: relative;
+  display: inline-grid;
+  grid-template-rows: min-content;
+  padding: var(--pf-c-tile--PaddingTop) var(--pf-c-tile--PaddingRight) var(--pf-c-tile--PaddingBottom) var(--pf-c-tile--PaddingLeft);
+  text-align: center;
+  background-color: var(--pf-c-tile--BackgroundColor);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    pointer-events: none;
+    content: "";
+    border: var(--pf-c-tile--before--BorderWidth) solid var(--pf-c-tile--before--BorderColor);
+    border-radius: var(--pf-c-tile--before--BorderRadius);
+  }
+
+  // stylelint-disable
+  &[disabled] {
+    pointer-events: none;
+    cursor: not-allowed;
+
+    --pf-c-tile--BackgroundColor: var(--pf-c-tile--disabled--BackgroundColor);
+    --pf-c-tile__title--Color: var(--pf-c-tile--disabled__title--Color);
+    --pf-c-tile__body--Color: var(--pf-c-tile--disabled__body--Color);
+
+    &::before {
+      border: 0;
+    }
+
+    .pf-c-tile__header.pf-m-stacked .pf-c-tile__icon {
+      --pf-c-tile__icon--Color: var(--pf-c-tile--disabled__header--m-stacked__icon--Color);
+    }
+  }
+
+  &:hover {
+    &::before {
+      --pf-c-tile--before--BorderColor: var(--pf-c-tile--before--hover--BorderColor);
+    }
+
+    cursor: pointer;
+
+    --pf-c-tile__title--Color: var(--pf-c-tile--hover__title--Color);
+    --pf-c-tile__icon--Color: var(--pf-c-tile--hover__icon--Color);
+
+    .pf-c-tile__header.pf-m-stacked .pf-c-tile__icon {
+      --pf-c-tile__icon--Color: var(--pf-c-tile--hover__header--m-stacked__icon--Color);
+    }
+  }
+
+  &.pf-m-selected,
+  &:focus {
+    &::before {
+      --pf-c-tile--before--BorderWidth: var(--pf-c-tile--before--m-selected--BorderWidth);
+      --pf-c-tile--before--BorderColor: var(--pf-c-tile--before--m-selected--BorderColor);
+    }
+
+    cursor: pointer;
+
+    --pf-c-tile__title--Color: var(--pf-c-tile--m-selected__title--Color);
+    --pf-c-tile__icon--Color: var(--pf-c-tile--m-selected__icon--Color);
+
+    .pf-c-tile__header.pf-m-stacked .pf-c-tile__icon {
+      --pf-c-tile__icon--Color: var(--pf-c-tile--m-selected__header--m-stacked__icon--Color);
+    }
+  }
+
+  &.pf-m-display-lg {
+    .pf-c-tile__header.pf-m-stacked {
+      .pf-c-tile__icon {
+        width: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--Width);
+        height: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--Height);
+
+        --pf-c-tile__icon--FontSize: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--FontSize);
+
+        img {
+          width: 100%;
+          height: 100%;
```

Comment:
this can create a skewed image, should be `max-width/height`, but what if it's an `svg`? We could just start with supporting icon fonts and react-icons SVGs, which will accept the `font-size` declaration, then create a follow-up to investigate other image formats.